### DISCUSSION TITLE
Configuration portail ENEDIS pour HA-Linky via Conso API ...

### DISCUSSION THREAD

**Discussion by loops21** [bokub/conso-api#64] [Aide]
Bonjour,

Je suis en train d'essayer de configurer mon portail ENEDIS en suivant le tuto présent dans la doc HA LINKY.

J'ai créé un compte sur le portail ENEDIS et ajouté mon compteur.
J'ai activé les "Enregistrement de la consommation horaire" et "Collecte de la consommation horaire" (puis j'ai patienté le temps que tout soit activé).

![image](https://github.com/user-attachments/assets/621393f1-3edd-4513-8ecf-c3e1483087b1)

Sauf erreur de ma part, je dois maintenant cliquer sur le bouton bleu se trouvant sur le site https://conso.boris.sh/

![image](https://github.com/user-attachments/assets/fc6a13a5-26ac-4d14-8b01-f119ce2cbffb)

J'arrive sur le portail ENEDIS, je vois bien mon compteur Linky mais il est grisé et quand je le survole j'ai le message "Compteur incompatible"

![image](https://github.com/user-attachments/assets/a9b99f45-971a-4d23-b875-4d80fc5f6145)

J'ai loupé un truc ou coquille chez ENEDIS ? 

Merci d'avance pour votre aide,

JC

**Comment by bokub** [comment]
Hello,

Tu trouveras ta réponse dans #62

**Comment by loops21** [comment]
Merci Beaucoup !
